app/src/main/res/layout/quest_answer_fragment_content.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_above="@+id/buttonPanel"
+    android:layout_marginStart="-9dp"
+    android:layout_marginEnd="-9dp"
+    android:layout_marginTop="-8dp"
+    android:layout_marginBottom="-8dp"
+    android:background="#00000000">
+    <!-- If I do not set any background color here, the layout will leave a few pixels free at the
+ bottom on my Android 6.0 emulator; so, setting to transparent-->
+
+    <LinearLayout
+        android:id="@+id/bottomSheet"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:elevation="16dp"
+        android:orientation="vertical"
+        app:layout_behavior="android.support.design.widget.BottomSheetBehavior"
+        app:behavior_peekHeight="@dimen/quest_bottom_sheet_peek_height"
+        android:background="@android:drawable/dialog_holo_light_frame"
+        >
+
+        <include layout="@layout/quest_answer_fragment_title_bar"/>
+
+        <android.support.v4.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <LinearLayout
+                android:id="@+id/content"
+                android:layout_height="wrap_content"
+                android:layout_width="match_parent"
+                android:orientation="vertical"
+                android:paddingLeft="16dp"
+                android:paddingRight="16dp"
+                android:paddingTop="8dp"
+                android:paddingBottom="8dp"
+                >
+
+            </LinearLayout>
+
+        </android.support.v4.widget.NestedScrollView>
+
+    </LinearLayout>
+
+</android.support.design.widget.CoordinatorLayout>
```

Comment:
Where is the difference between this and the `land`-version? I think it would make sense to add an explanatory commment here (also regarding the `elevation`-workaround with the negative margins, as it may not be apparent at first sight what is happening there).